definitions/json/CaseEventToFields.json

```diff
@@ -5033,6 +5032,170 @@
     "RetainHiddenValue": "Yes",
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "et3Notification",
+    "CaseFieldID": "et3NotificationDocCollection",
+    "DisplayContext": "MANDATORY",
```

Comment:
This should be COMPLEX - the fields part of the complexType should then be listed on EventToComplexTypes